packages/core/src/tenants/Libraries.ts

```diff
@@ -24,6 +25,7 @@ export default class Libraries {
   verificationStatuses = createVerificationStatusLibrary(this.queries);
   domains = createDomainLibrary(this.queries);
   quota = createQuotaLibrary(this.queries, this.cloudConnection, this.connectors);
+  logtoConfigs = createLogtoConfigLibrary(this.queries);
```

Comment:
https://github.com/logto-io/logto/blob/master/packages/core/src/tenants/Tenant.ts#L58

duplicate definition should use `tenantContext.logtoConfigs` instead.